networking/podenv.go

```diff
@@ -247,8 +247,9 @@ func loadUserNets(localConfig string, netsLoadList common.PrivateNetList) ([]act
 		}
 
 		// "default" is slightly special
```

Comment:
This comment has been there for a while. IMHO it can be removed as it doesn't really say anything